scripts/ci/gitlab/pipeline/check.yml

```diff
@@ -35,6 +35,19 @@ test-dependency-rules:
   script:
     - ./scripts/ci/gitlab/ensure-deps.sh
 
+test-rust-features:
+  stage:                           check
+  extends:
+    - .kubernetes-env
+    - .test-refs-no-trigger-prs-only
+  allow_failure:                   true
+  script:
+    - git clone
+        --depth=1
+        --branch="$PIPELINE_SCRIPTS_TAG"
+        https://github.com/paritytech/pipeline-scripts
```

Comment:
It how uses the script from the pipeline-scripts repo. Please have another look @alvicsam

Reply:
LGTM